docker-compose.yml

```diff
@@ -0,0 +1,22 @@
+version: '3'
+
+services:
+    database:
+        image: mysql:5.7
+        environment:
+            MYSQL_ROOT_PASSWORD: letmein™
+        restart: "always" 
+        ports:
+            - "33060:3306"
+        volumes:
+            - "C:/DockerData/MySMS:/var/lib/mysql"
+    
+    www:
+        build:
+            context: .
+            dockerfile: ./Dockerfile
+        depends_on:
+            - "database"
+        ports:
+            - "3000:3000"
+        command: bundle exec rails s
```

Comment:
Just 2 minor formatting things:

1. Looks like this file is using larger tab widths than other files ? might need to be changed to be consistent
2. Are you able to enable "add trailing newline" to your editor config ? also for consistency and is required for work in JR plus 🙂

Reply:
Fixed... will have to stop using multiple editors...